action.yml.erb

```diff
@@ -15,6 +15,10 @@ inputs:
     description: 'Publishes posts with a future date.'
     required: false
     default: true
+  build_revision:
+    description: 'The SHA-1 of the git commit for which the build is running. Default to GITHUB_SHA when not provided.'
+    required: false
+    default:
```

Comment:
I think this needs?

```suggestion
    default: true
```

Reply:
hmmm default should be the commit `SHA`? wondering if there is a variable for that... 🤔

Reply:
No that's the [default value](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault). I am leaving it empty because I don't think I can get an environment variable in there.

That means when it's empty (or not defined by someone), we fallback to `GITHUB_SHA` in the entrypoint script. That will work for anyone using the dynamic workflow or building on a branch/pull request trigger. There may be exotic triggers where `GITHUB_SHA` does not make any sense which is the only reason I introduced an input.

Reply:
For the token you use `default: ${{ github.token }}`, so perhaps here you could use `${{ github.sha }}` and get it from GitHub Actions? (also hi :wave: i started following this repo)

Reply:
Yep what @parkr said. Also 👋!

Reply:
Fixed in https://github.com/actions/jekyll-build-pages/pull/8. Sorry for the merge, I have fat fingers today 😢.